tokenmagic/module/autoTemplate/dnd5e.js

```diff
@@ -157,8 +157,20 @@ export class AutoTemplateDND5E {
 		if (!item.hasDamage) {
 			return false;
 		}
-		let dmgSettings = categories[item.data.data.damage.parts[0][1]] || {};
-		let config = dmgSettings[template.data.t];
+
+		let config, dmgSettings;
+
+		// some items/spells have multiple damage types
+		for (const [_, dmgType] of item.data.data.damage.parts) {
+			dmgSettings = categories[dmgType] || {};
+			config = dmgSettings[template.data.t];
+
+			// if we encounter a damage type that doesn't have a preset (physical types by default), skip
+			// so we match the first damage type that presents a valid fx preset
```

Comment:
This comment doesn't appear to match the action taken here, I suspect this was written for an earlier version of the patch?

Reply:
It's not outdated, it just describes the behavior in a roundabout way. I reworded it, lmk if it's still confusing.